### ISSUE TITLE
A11y: Priority badges doesn't pass the minimum contrast requirements

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/vacant-lots-proj#339]
## Description

The badges `High Priority` , ` Medium Priority` & `Low Priority` don't pass the text minimum contrast requirements. If we are sticking to the current colors, let's make the text black so that it passes.

<details>
<summary>Example</summary>

<img width="570" alt="Screenshot 2024-02-12 at 20 08 49" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/570d019e-0107-4741-a54f-06eb03da387d">

</details>

## Recommendation
- Make text black (easiest. It matches the Filter page)
- Or change the badge background color to a darker shade and make sure that the contrast between the white text & background is 4.5:1.

**Comment by thansidwell** [comment]
@marvieqa I made a new design for the badges that does pass contrast requirements. Here it is. https://www.figma.com/file/NAFkgq34abW6uJ0R7PW24T/Prototype---Clean-%26-Green-Philly?type=design&node-id=3019%3A13737&mode=design&t=hRcztEw6fOq1MNmn-1

![Image](https://github.com/CodeForPhilly/vacant-lots-proj/assets/1965986/7776dad1-601f-491c-a17b-085ad8a59bad)

**Comment by Moylena** [comment]
New design fixes this. Will close and open another ticket to create the new one to reflect the new design

**Comment by marvieqa** [comment]
resolved with the new design and is live in Staging.
